docs/en/offerings/spc/setup/traffic_segregation.md

```diff
@@ -30,3 +32,13 @@ Example DNS entry
 ```
 mystudio-staging.shotgunstudio.com   CNAME   vpce-048447456a4f57e14-1j3wh50q.vpce-svc-0b054415458f57634.us-west-2.vpce.amazonaws.com
 ```
+
+## Validation
+
+Verify that your site resolve to IPs in your AWS VPC block.
+
+```
+nslookup mystudio-staging.shotgunstudio.com
+```
+
+Try to access your test site from inside your office ie https://mystudio-staging.shotgunstudio.com
```

Comment:
since you switched staging to test above in the doc, might as well change it here too

Reply:
I will keep staging for now.